docs/modules/general/pages/bootstrapping.adoc

```diff
@@ -0,0 +1,567 @@
+
+Depending on the content type and chosen expose method, different adjustments have to be made to ensure correct content bootstrapping. 
+
+=== Content Exposed Using Angular Method
+IMPORTANT: **It is NOT recommended** to use the Angular expose method. Doing so restrains the platform from using many frameworks with different versions, which means that all Microfrontends would need to use the same technology and version.
+
+When exposing modules and Remote Components using the Angular method, there are two requirements.
+
+. Webpack configuration needs to expose the module or component file directly. It means that instead of pointing to a **main.ts** file that is bootstrapping the content, **module.ts** and **component.ts** files should be used directly.
+. The **main.ts** file of Microfrontend's modules (content type) needs to bootstrap the module asynchronously.
++
+**main.ts** content should look similar to:
+[source,typescript]
+import('./bootstrap').catch(err => console.error(err));
++
+**bootstrap.ts** content should look similar to (or any custom bootstrap of your Microfrontend):
++
+[source,typescript]
+....
+import { AppModule } from './app/app.module';
+import { environment } from './environments/environment';
+import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
+import { enableProdMode } from '@angular/core';
+
+if (environment.production) {
+  enableProdMode();
+}
+
+platformBrowserDynamic().bootstrapModule(AppModule)
+  .catch(err => console.error(err));
+....
+
+=== Content Exposed Using Webcomponent Method
+// TODO: Correct link to exact resource for @onecx/angular-webcomponents lib
+xref:implementation-details/mfe-content-loading/webcomponents.adoc[Loading mechanism for modules and Remote Components exposed using Webcomponent method] requires that content is adapted to using Web Components Custom Elements. For both content types, bootstrapping logic needs to register Custom Element in the Custom Elements Registry.
+
+// TODO: Correct link to exact resource for Angular issues
+NOTE: For Angular based content, apart from Custom Element registration it is also required to cover a few Angular related issues. For more information on those issues, please refer xref:implementation-details/mfe-content-loading/webcomponents.adoc[here]
+
+// TODO: Correct link to exact resource for @onecx/angular-webcomponents lib
+It is recommended to use xref:implementation-details/mfe-content-loading/webcomponents.adoc[@onecx/angular-webcomponents] library to bootstrap Microfrontend's content using its exposed methods. This library will be used in the examples throughout this document.
+
+==== Module Bootstrapping and Configuration
+The following requirements need to be fullfield when exposing modules using Webcomponent method:
```

Comment:
The following requirements need to be ~~fullfield~~ **fulfilled** when exposing modules using **the** Webcomponent method: